dbms/src/Storages/StorageGenerate.cpp

```diff
@@ -1,491 +0,0 @@
-#include <Storages/IStorage.h>
```

Comment:
Lets try to preserve history for a file. Probably mv file in a separate commit can help.

Reply:
I already made `git mv` but then the history is lost.